pkg/security/apiserver/securitycontextconstraints/provider.go

```diff
@@ -278,7 +279,7 @@ func (s *simpleProvider) ValidatePodSecurityContext(pod *api.Pod, fldPath *field
 }
 
 // Ensure a container's SecurityContext is in compliance with the given constraints
-func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, container *api.Container, fldPath *field.Path) field.ErrorList {
+func (s *simpleProvider) ValidateContainerSecurityContext(pod *corev1.Pod, container *corev1.Container, fldPath *field.Path) field.ErrorList {
```

Comment:
this looks off.  When you come in from admission, you're going to have an internal pod, right?